dashboard/agent.py

```diff
@@ -83,9 +83,10 @@ def __init__(self,
             assert self.ppid > 0
             logger.info("Parent pid is %s", self.ppid)
         self.server = aiogrpc.server(options=(("grpc.so_reuseport", 0), ))
+        grpc_ip = "127.0.0.1" if self.ip == "127.0.0.1" else "0.0.0.0"
```

Comment:
I wonder if we should just do `platform.system() == "Darwin"` instead?

Reply:
If node ip is 127.0.0.1, then we are in the local/single node mode regardless if it's linux, mac or windows, and we should bind to 127.0.0.1

And we have a central place to decide the node ip which is `services.get_node_ip_address` so if we want to do the same thing for other platform, we just need to change that one place. In fact, both windows and mac have the popup issue.

Reply:
We should keep the dashboard agent listen ip the same as raylet, if the raylet listens to 127.0.0.1, then the dashboard agent needs to listen to 127.0.0.1

Reply:
Yes, ralyet has the same logic so they are consistent.